src/components/pages/creation.tsx

```diff
@@ -1,5 +1,99 @@
 function Creation(): JSX.Element {
-  return <div>Creation</div>;
+  return (
+    <div style={{ textAlign: 'left', paddingLeft: '2.5vw' }}>
+      Creation and Deletion
+      <br />
```

Comment:
We shouldn't need to use breaks. Instead wrap the text in a paragraph tag.

Reply:
should i use a paragraph tag for each sentence since each is on a new line? or can i use breaks within a paragraph?